device/accton/x86_64-accton_as7712_32x-r0/Accton-AS7712-32X/port_config.ini

```diff
@@ -1,33 +1,33 @@
-# name          lanes                 alias
-Ethernet0       49,50,51,52           hundredGigE1
-Ethernet4       53,54,55,56           hundredGigE2
-Ethernet8       57,58,59,60           hundredGigE3
-Ethernet12      61,62,63,64           hundredGigE4
-Ethernet16      65,66,67,68           hundredGigE5
-Ethernet20      69,70,71,72           hundredGigE6
-Ethernet24      73,74,75,76           hundredGigE7
-Ethernet28      77,78,79,80           hundredGigE8
-Ethernet32      33,34,35,36           hundredGigE9
-Ethernet36      37,38,39,40           hundredGigE10
-Ethernet40      41,42,43,44           hundredGigE11
-Ethernet44      45,46,47,48           hundredGigE12
-Ethernet48      81,82,83,84           hundredGigE13
-Ethernet52      85,86,87,88           hundredGigE14
-Ethernet56      89,90,91,92           hundredGigE15
-Ethernet60      93,94,95,96           hundredGigE16
-Ethernet64      97,98,99,100          hundredGigE17
-Ethernet68      101,102,103,104       hundredGigE18
-Ethernet72      105,106,107,108       hundredGigE19
-Ethernet76      109,110,111,112       hundredGigE20
-Ethernet80      17,18,19,20           hundredGigE21
-Ethernet84      21,22,23,24           hundredGigE22
-Ethernet88      25,26,27,28           hundredGigE23
-Ethernet92      29,30,31,32           hundredGigE24
-Ethernet96      113,114,115,116       hundredGigE25
-Ethernet100     117,118,119,120       hundredGigE26
-Ethernet104     121,122,123,124       hundredGigE27
-Ethernet108     125,126,127,128       hundredGigE28
-Ethernet112     1,2,3,4               hundredGigE29
-Ethernet116     5,6,7,8               hundredGigE30
-Ethernet120     9,10,11,12            hundredGigE31
-Ethernet124     13,14,15,16           hundredGigE32
+# name          lanes                 alias             index  speed
+Ethernet0       49,50,51,52           hundredGigE1      0      100000
```

Comment:
check you index, it should match the front panel port number.

Reply:
This PR will essentially would help in correct port utilization calculation and port-channel speed calculation and also avoid CLI script failures when port-channel exists on these platforms. In my opinion, changing 0-based index to 1-based index for AS7712 can be taken up separately. This would require additional changes it widens scope of this PR.